sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/reflection/Reflect.scala

```diff
@@ -50,4 +54,32 @@ object Reflect {
     }
 
   }
+
+  private implicit val stringArrayOrdering: Ordering[Array[String]] =
+    Ordering.fromLessThan(util.Arrays.compare[String](_, _) < 0)
+
+  implicit val methodOrdering: Ordering[Method] =
+    Ordering.by((m: Method) => (m.getName, m.getReturnType.getName, m.getParameterTypes.map(_.getName)))
```

Comment:
Orthogonal to main change in this PR. 
I merged ReflectionUtils with Reflect class.